srcalc/src/main/java/gov/va/med/srcalc/web/view/admin/EditRiskModel.java

```diff
@@ -27,23 +183,34 @@
     // if not.
     private RiskModel importedModel = null;
     
-    protected EditRiskModel(final RiskModel rm)
+    protected EditRiskModel(final RiskModel rm, final List<Specialty> applSpecialty)
     {
         fRiskModel = rm;
         modelName = rm.getDisplayName();
```

Comment:
Private variables should be named with a lower-cased f at the front of the variable name.